### ISSUE TITLE
Admin stats page display

### ISSUE THREAD

**Issue by jonibar** [psu-libraries/scholarsphere-3#912]
When scrolling the admin statistics page, the background color turns grey making it difficult to read the list of Deposits by Users.

![screen shot 2017-06-06 at 3 24 54 pm](https://user-images.githubusercontent.com/2422369/26847855-80414558-4acc-11e7-8822-641f4f904591.png)

blocked by #1175

**Comment by carolyncole** [comment]
Double check to make sure this is fixed after fixing #1175

**Comment by carolyncole** [comment]
@mtribone believes this could be related to CSS positioning.

**Comment by awead** [comment]
I can't seem to reproduce this. After faking some data locally, with 1000 users, it seems to render fine:
![admin.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/55098a50f74dcc53007cd9d5/125516c66e82c728ace21e0d46e990972a8cc1e5cdf9ad4cae58f3806b1c65267d01a72d2180b72ae9112b55575e47ac415e5842fcaf8e39baa47e21d40854b088270a36d43703adbf881eba02bf20884e8cbebd884dc8fa70.png)
However, we really won't be able to tell until #1175 is merged into production and the page renders correctly.
